client/src/components/ContestList/ContestList.tsx

```diff
@@ -6,37 +6,52 @@ import Avatar from '@material-ui/core/Avatar';
 import Typography from '@material-ui/core/Typography';
 import Container from '@material-ui/core/Container';
 import Button from '@material-ui/core/Button';
-import useStyles from './useStyles'
+import useStyles from './useStyles';
+import { Contest } from '../../interface/User';
 import LionTatoo from "../../Images/lionTatoo.png";
 
+interface Props {
+    userContests: Contest[];
+}
 
-export default function ContestList(): JSX.Element {
+export default function ContestList({ userContests }: Props): JSX.Element {
     const classes = useStyles();
 
     return (
         <>
-            <List className={classes.list}>
-                <ListItem>
-                    <Avatar className={classes.avatar} alt="Test Contest Image" variant="square" src={LionTatoo} />
-                    <Grid className={classes.grid} container >
-                        <ListItemText
-                            primary={
-                                <>
-                                    <Typography className={classes.typography}>Lion tatoo concept in minimal style</Typography>
-                                </>
-                            }
-                            secondary={
-                                <>
-                                    <Typography>Looking for cool simplicity ideas of a lion.</Typography>
-                                    <Container className={classes.container}>
-                                        <Button className={classes.button}>$150</Button>
-                                    </Container>
-                                </>
-                            }
-                        />
-                    </Grid>
-                </ListItem>
-            </List>
+            {/* Write ternary operator checking to see if any data has been recieved, otherwise return default message */}
+            {userContests.length > 0 ? (
+
+                <List className={classes.list}>
+                    {/* map through given data and alter the return inside of the map */}
+                    {userContests.map(contest => {
+                        return (
+                            <ListItem key={contest.userId}>
```

Comment:
Please use a different key here, as the `userId` will be the same for every item.

Reply:
good catch, thank you, will be updating changes shortly and merging.